content/r/concepts/data-frames/data-frames.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'Data Frames'
+Description: 'Data Frames are objects that store data into two dimensions of rows and columns.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data Structures'
+  - 'Data Types'
+  - 'Objects'
+  - 'Tables'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+**Data Frames** are objects that store data into a table with two dimensions represented by columns and rows. The columns are the different characteristics and the rows are instances of a set of characteristics. Data frames are useful for storing different [data types](https://www.codecademy.com/resources/docs/r/data-types).
+
+## Creating Data Frames
+
+Data Frames can be created using the `data.frame()` function.
+
+### Syntax
+
+```pseudo
+data.frame(arguments)
+```
+
+The `arguments` are the list of vectors with the values for each column of the data frame.
```

Comment:
Do we have an issue for adding vectors to R data types? If not, maybe this would be a good addition. Just a thought to eventually link here

Reply:
No issue for vectors in R right now but I think vectors are a kind of data structure, not type. If we made an issue for this entry to be made, I'd recommend it be created as its own concept entry (similar to other entries about structures like arrays or strings).